src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/index.tsx

```diff
@@ -42,7 +42,7 @@ const IMAGE_INFO_UNAVAILABLE = '-1';
 
 type DefaultProps = {
   data: {
-    images: Array<Image>;
+    images: Array<Image | null>;
```

Comment:
Can you have an array of nulls?

Reply:
yes. I had this problem recently and noticed that we also checked for this before in the function `isValidImage`